load-generator/README.md

```diff
@@ -49,7 +41,7 @@ The following is a list of additional optional command line arguments applicable
 
 ### StatsD Metrics Load Test Sample Command,
 ```
-./gradlew :load-generator:run --args="metric --metricCount=100 --datapointCount=10 --observationInterval=1000 -u=localhost:8125 -d=statsd"
```

Comment:
observationInterval=1000 should we have observation interval configurable.

Reply:
I think as sample app, providing configurable observationInterval is good, but in load generator context and how it is used through test-framework; I think it would be unused functionality and could create problem as well.